dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/weld/BeanDeploymentArchiveImpl.java

```diff
@@ -880,6 +881,11 @@ public void addEjbDescriptor(EjbDescriptor<?> ejbDescriptor) {
 
         this.ejbDescriptors.add(ejbDescriptor);
         Class<?> beanClass = ejbDescriptor.getBeanClass();
+
+        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+            Tr.debug(tc, "addBeanDeploymentArchive: EjbDescriptor's beanClass " + beanClass.getName() + " " + System.identityHashCode(beanClass));
+        }
```

Comment:
Copyright date

Reply:
https://github.com/OpenLiberty/open-liberty/pull/30969 will be going in first, I'll avoid touching the copyright here to avoid a merge conflict